.gitmodules

```diff
@@ -157,3 +157,6 @@
 [submodule "contrib/openldap"]
 	path = contrib/openldap
 	url = https://github.com/openldap/openldap.git
+[submodule "contrib/SIMDxorshift"]
+	path = contrib/SIMDxorshift
+	url = https://github.com/lemire/SIMDxorshift
```

Comment:
I wonder what is better - SIMD xorshift by Daniele or SIMD pcg by the same author.